backend_new/src/controllers/listing.controller.ts

```diff
@@ -68,6 +76,32 @@ export class ListingController {
     );
   }
 
+  @Post()
+  @ApiOperation({ summary: 'Create listing', operationId: 'create' })
+  @UseInterceptors(ClassSerializerInterceptor)
+  @UsePipes(new ListingCreateUpdateValidationPipe(defaultValidationPipeOptions))
+  @ApiOkResponse({ type: Listing })
+  async create(@Body() listingDto: ListingCreate): Promise<Listing> {
+    return await this.listingService.create(listingDto);
+  }
+
+  @Delete()
+  @ApiOperation({ summary: 'Delete listing by id', operationId: 'delete' })
+  @UsePipes(new ValidationPipe(defaultValidationPipeOptions))
+  async delete(@Body() dto: IdDTO): Promise<SuccessDTO> {
+    return await this.listingService.delete(dto.id);
+  }
```

Comment:
Your preference makes sense to me. I'm guessing in a follow-up PR?

Reply:
yeah either in a follow up pr or in at some point kinda deal, I don't think its pressing to get that update wrapped into the prisma change over since functionally it doesn't affect much.

Going to resolve this comment for now, and spun up:  https://github.com/bloom-housing/bloom/issues/3626 to capture that work